workflows/icons/anthos.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.2.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.0" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 300 280" enable-background="new 0 0 300 280" xml:space="preserve">

```

Comment:
can you check this?

Reply:
also, rename the icon name to `bank-of-athos`

Reply:
This one is .svg format we have given link at 
in bank-of-anthos.chartserviceversion.yaml
    - url: https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/workflows/icons/anthos.svg